rfp-proposals/rfp-proposal-filemeta.io.md

```diff
@@ -0,0 +1,147 @@
+
+The first part will be done by one designer while discussed by all the team members.
+
+When the second part is finished,we'll deploy a `Mongodb Sharding Cluster`,and run one or more `Filecoin Node` which has the ability to process data in real time.Depending on the usage scenario,it will import `Filecoin Blockchain Data` into database or message queue in real time,which is still under consideration.
+
+The second part will be done by two backend programmers.
+
+For greater development efficiency,we may not provide hosted front or backend programs during this period.
+
+### Milestone 3
+
+In this period,we'll follow the design plan to complete the development of the front and backend of the explorer.Finally,it will be hosted and the loading speed will be optimized.
+
+According to the schedule,the explorer at this stage will be compatible with Filecoin Beta Testnet or the latest test network at that time.
+
+When this phase is complete,we'll release the Beta version of `filemeta.io` and host it on our severs for `https` access.At this point,we will still accept opinions from community and `Filecoin Team` to optimize and improve `filemma.io` appropriately.
+
+This phase will be done by a frontend programmer and a backend programmer.
+
+### Milestone 4
+
+In this phase,we will complete all the unfinished work of the project,including API documentation,more adequate testing and optimization,adding the codebase to [Filecoin Shipyard](https://github.com/filecoin-shipyard),and so on.
+
+This phase will be done by a frontend programmer and two backend programmers.
+
+### Milestone 5
+
+Taking third party data storage solutions,high availablity and big data architectures for advantages make our backend  much more complicated than other solutions,which may require more used servers and more labor costs.
+
+Over the next year,we will continue to maintain and upgrade the explorer to ensure stable operation.
+
+Our team will be jointly responsible for this phase.
+
+## Total Budget Requested
+
+$80000
+
+## Maintenance and Upgrade Plans
+
+We promise to maintain and upgrade `filemeta.io` for at least 1 year.
+
+In fact,our team plans to join `Filecoin` community full time for developing more applications.If all goes well,we may be able to maintain and develop all the time.
+
+# Team
+We are a small entrepreneurial team in China.We joined EOS community with [EOSTEA](https://eostea.io) in November 2017.Since then,we had maintained the largest Chinese developer community of EOS `eosfans.io`. We also developed EOS blockchain explorers,such as `eosmonitor.io`(the first explorer of EOS community) and `eosmeta.io`. We withdrawed from EOS community,and closed `eosmonitor.io` & `eosmeta.io` in October 2018,duing to the complicated relationships in the community and the huge maintenance cost of the project.
```

Comment:
Do you have any screenshots of what eosmonitor.io and eosmeta.io before they were shut down?

Reply:
We couldn't run the previous project because there was no database, but I found some images from `Google Images`.

### eosmonitor.io

![3](https://user-images.githubusercontent.com/7932644/67415087-ed175500-f5f6-11e9-9295-5583cb0ed548.png)

### eosmeta.io

![10](https://user-images.githubusercontent.com/7932644/67415162-189a3f80-f5f7-11e9-83c8-52f71e5bd76c.png)

### eosmeta.io (weapp/mobile version)
![4](https://user-images.githubusercontent.com/7932644/67415423-9b22ff00-f5f7-11e9-81bf-f74fc799803f.png)
![5](https://user-images.githubusercontent.com/7932644/67415432-9e1def80-f5f7-11e9-9c4d-b2f2bfc6f0d9.png)
![6](https://user-images.githubusercontent.com/7932644/67415441-a118e000-f5f7-11e9-8fba-0191cc7e2c42.png)
![7](https://user-images.githubusercontent.com/7932644/67415447-a413d080-f5f7-11e9-986c-428f69b47e87.png)
![11](https://user-images.githubusercontent.com/7932644/67415463-a7a75780-f5f7-11e9-996d-a251878de907.png)